hack/e2e/run.sh

```diff
@@ -82,6 +83,8 @@ HELM_CT_TEST=${HELM_CT_TEST:-"false"}
 CHART_TESTING_VERSION=${CHART_TESTING_VERSION:-3.7.1}
 CLEAN=${CLEAN:-"true"}
 
+METRICS=${METRICS:-"false"}
```

Comment:
Can we consider naming this something like `COLLECT_METRICS`, to make it more apparent what it does?